app/globals.css

```
@@ -78,6 +81,24 @@ body {
   }
 }
 
+@keyframes colorCycle {
```

Stylelint (17.12.0)

[error] Expected keyframe name "colorCycle" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)